downstream/modules/platform/proc-attaching-subscriptions.adoc

```diff
@@ -5,10 +5,7 @@
 = Attaching your {PlatformName} subscription
 
 [role="_abstract"]
-You *must* have valid subscriptions attached before installing {PlatformName}. Attaching an {PlatformNameShort} subscription enables {HubName} repositories.
-
-A valid subscription needs to be attached to the {HubName} node only. Other nodes do not need to have a valid subscription attached, even if the `[automationhub]` group is blank, given this is done at the `repos_el` role level and that this role is run on both `[automationcontroller] and `[automationhub]` hosts.
-
+You *must* have valid subscriptions attached on all nodes before installing {PlatformName}. Attaching your {PlatformNameShort} subscription allows you to access subcription-only resources, such as {HubName} repositories.
```

Comment:
There is no longer "automation hub repository" and we only have a repo for AAP.

Also, I'm not sure what the instruction has been, when using bundle installer in the disconnected envionment. But it looks like that part hasn't changed in this PR, so I think we're good?  (What we provide in bundle is what you'd get from AAP repo and customers still need to configure in a way they get RHEL rpms from somewhere).